src/content/developers/docs/oracles/index.md

```diff
@@ -1,314 +1,86 @@
-2. An off-chain service has subscribed (usually using something like the JSON-RPC `eth_subscribe` command) to these specific logs.
-3. The off-chain service proceeds to do some tasks as defined by the log.
-4. The off-chain service responds with the data requested in a secondary transaction to the smart contract.
-
-This is how to get data in a 1 to 1 manner, however to improve security you may want to decentralize how you collect your off-chain data.
-
-The next step might be to have a network of these nodes making these calls to different APIs and sources, and aggregating the data on-chain.
-
-[Chainlink Off-Chain Reporting](https://blog.chain.link/off-chain-reporting-live-on-mainnet/) (Chainlink OCR) has improved on this methodology by having the off-chain oracle network communicate with each other, cryptographically sign their responses, aggregate their responses off-chain, and send only one transaction on-chain with the result. This way, less gas is spent, but you still get the guarantee of decentralized data since every node has signed their part of the transaction, making it unchangeable by the node sending the transaction. The escalation policy kicks in if the node doesn't transact, and the next node sends the transaction.
-
-## Usage {#usage}
-
-Using services like Chainlink, you can reference decentralized data on-chain that has already been pulled from the real world and aggregated. Sort of like a public commons, but for decentralized data. You can also build your own modular oracle networks to get any customized data you're looking for. In addition, you can do off-chain computation and send information to the real world as well. Chainlink has the infrastructure in place to:
-
-- [Get crypto price feeds in your contract](https://chain.link/solutions/defi)
-- [Generate verifiable random numbers (useful for gaming)](https://chain.link/solutions/chainlink-vrf)
-- [Call external APIs](https://docs.chain.link/docs/request-and-receive-data)
-  – One novel use of this is [Checking wBTC reserves](https://cointelegraph.com/news/1b-in-wrapped-bitcoin-now-being-audited-using-chainlink-s-proof-of-reserve)
-
-Here is an example of how to get the latest ETH price in your smart contract using a Chainlink price feed:
-
-### Chainlink Data Feeds {#chainlink-data-feeds}
-
-```solidity
-pragma solidity ^0.6.7;
-
-import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
-
-contract PriceConsumerV3 {
-
-    AggregatorV3Interface internal priceFeed;
+Giving smart contracts the ability to execute based on real-world inputs vastly extends the value of decentralized applications. For instance, decentralized prediction markets rely on oracles to provide information about outcomes with which they can validate predictions from users. If Alice bets 20 ETH on who becomes the next U.S. President, the prediction-market dapp needs an oracle to confirm election results and determine if Alice is eligible for a payout. 
```

Comment:
```suggestion
Giving smart contracts the ability to execute based on real-world inputs extends decentralized applications' value. For instance, decentralized prediction markets rely on oracles to provide information about outcomes with which they can validate user predictions. Suppose Alice bets 20 ETH on who will become the next U.S. President. In that case, the prediction-market dapp needs an oracle to confirm election results and determine if Alice is eligible for a payout.
```

Reply:
1. I wonder if the example about Alice is too in the weeds for the introduction paragraphs.
2. The way this section ends feels a little abrupt. Making me feel more like removing it is a good option.

Reply:
> I wonder if the example about Alice is too in the weeds for the introduction paragraphs.

1. The original page already had this: "For example, prediction market dapps use oracles to settle payments based on events. A prediction market may ask you to bet your ETH on who will become the next president of the United States. They'll use an oracle to confirm the outcome and payout to the winners." The Alice example was to make the concept a bit clearer. 

2. The Alice example doesn't feel too technical in my opinion since we already describe oracles as sourcing information from the real-world for use on-chain. 

> The way this section ends feels a little abrupt. Making me feel more like removing it is a good option.

It does feel *brief*, but I think the intros shouldn't be longer than necessary. This one prepares the reader for the content that follows, so leaving it in might be a good idea.